src/systemd/sd-lldp.h

```diff
@@ -33,6 +33,87 @@ _SD_BEGIN_DECLARATIONS;
 typedef struct sd_lldp sd_lldp;
 typedef struct sd_lldp_neighbor sd_lldp_neighbor;
 
+#define SD_LLDP_MULTICAST_ADDR     { 0x01, 0x80, 0xc2, 0x00, 0x00, 0x0e }
```

Comment:
> What's the point of exposing this? I am pretty sure we should be as conservative as possible with what we expose.

Yeah, this alone isn't very useful as it is only one of the accepted mac addresses.

Instead it would be probably more useful for clients to have defines as 

 - SD_LLDP_ADDRESS_NEAREST_BRIDGE
 - SD_LLDP_ADDRESS_NEAREST_NON_TPMR_BRIDGE
 - SD_LLDP_ADDRESS_NEAREST_CUSTOMER_BRIDGE

to be used for determining the scope of propagation of the frame (i.e. if it was sent by a neighbor on the same physical link or not). But this isn't strictly necessary either.

I agree to unexport SD_LLDP_MULTICAST_ADDR.